examples/quickstart/src/main/java/quickstart/Quickstart.java

```diff
@@ -47,6 +47,7 @@ public static void main(String[] args) {
             Client client = builder.build();
 
             // Create a group
+            // For OAuth2 flow, ensure that you enable 'group' scopes otherwise the below group creation call would fail with HTTP 403 error.
```

Comment:
See line 102 in the README.md for instructions on how to regenerate examples

Reply:
Yeah, I have seen that before. But in this context here, are you suggesting this comment doesn't belong in here?

Reply:
Yes, I have seen that before. Are you suggesting the above comment doesn't belong there?

Reply:
My suggestion would be to leave it out, and add this to a section in the README.

That said, i was thinking this was the snippets class, so updates to this file wouldn't change the README anyway.  sorry about the confusion

Reply:
Any recommendations on which section in README should I add this to? I don't see a suitable section that this note could fit in. We could omit it perhaps as it is quite obvious?